docs/src/Modal.doc.js

```diff
@@ -528,6 +528,72 @@ function Example(props) {
   />
 );
 
+card(
+  <Example
+    name="Outside Event Exception Test"
+    description={`
+      Testing that outside event is working as expected with portals
+    `}
+    defaultCode={`
+function Example(props) {
+  const [showModal, setShowModal] = React.useState(false);
+  const [hasLoaded, setHasLoaded] = React.useState(false);
+  const [showFlyout, setShowFlyout] = React.useState(false);
+
+  const anchorRef = React.useRef();
+
+  const handleToggleModal = () => {
+    setShowModal(!showModal);
+    setHasLoaded(false);
+  }
+
+  return (
+    <Box marginLeft={-1} marginRight={-1}>
+      <Box padding={1}>
+        <Button
+          text="View Modal"
+          onClick={handleToggleModal}
+        />
+        {showModal && (
+          <Modal
```

Comment:
@emilyalwayss can we wrap `Modal` in a `<Layer />`? Otherwise, it'll show behind the code example in the docs:
![image](https://user-images.githubusercontent.com/127199/77769638-56b1ea00-7001-11ea-812c-7cd938c0dc59.png)